bin/osprey-mock-service.js

```diff
@@ -9,16 +9,18 @@ var morgan = require('morgan')
 var argv = require('yargs')
   .usage(
     'Generate an API mock server from a RAML definition.\n\n' +
-    'Usage: $0 -f [file] -p [port number] --cors'
+    'Usage: $0 -f [file] -p [port number] --cors --definition [definition uri]'
   )
   .demand(['f', 'p'])
   .describe('f', 'Path to the RAML definition')
   .describe('p', 'Port number to bind the proxy')
   .describe('cors', 'Enable CORS with the API')
+  .describe('definition', 'URI of raml definition')
```

Comment:
I think we shouldn't required an extra parameter here. Instead, we should build the URL based on the API baseUri of the RAML file provided by `-f`.